app_dart/test/request_handlers/get_repos_test.dart

```diff
@@ -2,39 +2,43 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:convert';
+// FOR REVIEW:
+// My understanding is that the single test in this file only checkes for literal match and not regExps match
+// after we update the get branch handler, is it okay if I delete this test?
 
-import 'package:cocoon_service/src/request_handlers/get_branches.dart';
-import 'package:cocoon_service/src/request_handling/body.dart';
-import 'package:test/test.dart';
+// import 'dart:convert';
 
-import '../src/datastore/fake_config.dart';
-import '../src/request_handling/request_handler_tester.dart';
+// import 'package:cocoon_service/src/request_handlers/get_branches.dart';
+// import 'package:cocoon_service/src/request_handling/body.dart';
+// import 'package:test/test.dart';
+
+// import '../src/datastore/fake_config.dart';
+// import '../src/request_handling/request_handler_tester.dart';
 
 void main() {
-  group('GetBranches', () {
-    late FakeConfig config;
-    late RequestHandlerTester tester;
-    late GetBranches handler;
-
-    setUp(() {
-      config = FakeConfig();
-      tester = RequestHandlerTester();
-      handler = GetBranches(
-        config,
-      );
-    });
-
-    test('returns branches matching regExps', () async {
-      config.flutterBranchesValue = <String>['flutter-1.1-candidate.1', 'master'];
-
-      final Body body = await tester.get(handler);
-      final Map<String, dynamic> result = await utf8.decoder
-          .bind(body.serialize() as Stream<List<int>>)
-          .transform(json.decoder)
-          .single as Map<String, dynamic>;
-
-      expect(result['Branches'], <String>['flutter-1.1-candidate.1', 'master']);
-    });
-  });
+//   group('GetBranches', () {
```

Comment:
umm I was thinking about deleting it because the test description says the test `returns branches matching regExps`, but I didn't see any regex matching in this test. My understanding is that it is setting flutterBranchesValue to be flutter-1.1-candidate.1, and return that value to check for flutter-1.1-candidate.1, which isn't very useful?

Reply:
Thanks! Then let's delete it